site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -296,10 +297,10 @@ function setPageBackgroundColor() {
 }
 ```
 
-The updated code adds an `onclick` event on the button, which triggers a [programmatically injected
-content script][24]. This turns the background color of the page the same color as the button. Using
-programmatic injection allows for user-invoked content scripts, instead of auto inserting unwanted
-code into web pages.
+The updated code adds a `click` event listener to the button, which triggers a [programmatically
+injected content script][24]. This turns the background color of the page the same color as the
+button. Using programmatic injection allows for user-invoked content scripts, instead of auto
```

Comment:
I it might be worth elaborating on why "auto inserted unwanted code" is bad.

Is the point here that extensions used to historically allow running always on every single page? Is it worth mentioning the permission that let them do that, and how it's discouraged? Could be good place to put an `{% Aside %}`.